src/pgen/cluster/agn_triggering.cpp

```diff
@@ -485,15 +528,34 @@ parthenon::TaskStatus
 AGNTriggeringFinalizeTriggering(parthenon::MeshData<parthenon::Real> *md,
                                 const parthenon::SimTime &tm) {
   auto hydro_pkg = md->GetBlockData(0)->GetBlockPointer()->packages.Get("Hydro");
-  const auto &agn_triggering = hydro_pkg->Param<AGNTriggering>("agn_triggering");
+  auto &agn_triggering = hydro_pkg->Param<AGNTriggering>("agn_triggering");
+  parthenon::Real cold_mass = 0.0; // Declare outside the if block
+  parthenon::Real total_mass = 0.0; // Declare outside the if block
+  if (agn_triggering.triggering_mode_ == AGNTriggeringMode::COLD_GAS) {
+    cold_mass = hydro_pkg->Param<Real>("agn_triggering_cold_mass"); // Assign here
+    total_mass = hydro_pkg->Param<Real>("agn_triggering_total_mass"); 
+
+
+    agn_triggering.inflow_cold_ = (cold_mass - hydro_pkg->Param<Real>("agn_triggering_prev_cold_mass")) / tm.dt; //Now this is the only place to calculate inflow rates
+    agn_triggering.inflow_tot_ = (total_mass - hydro_pkg->Param<Real>("agn_triggering_prev_total_mass")) / tm.dt; //Now this is the only place to calculate inflow rates
+
```

Comment:
Why are those new member variables required?
As far as I can tell these are temporary values only used in this function as you can calculate those from `total` and `prev_total`s

Reply:
because I need to compute the inflow rate and in Finalizetriggering function I compute inflow rate values and record the values in order to get them in output in agn_triggering.dat file (line 553 554)